lib/robots/dor_repo/accession/end_accession.rb

```diff
@@ -13,12 +13,7 @@ def perform_work
           next_dissemination_wf = special_dissemination_wf
           workflow_service.create_workflow_by_name(druid, next_dissemination_wf, version: current_version, lane_id:) if next_dissemination_wf.present?
 
-          # Note that this used to be handled by the disseminationWF, which is no longer used.
-          object_client.workspace.cleanup(workflow: 'accessionWF', lane_id:)
-
```

Comment:
don't make this call from the end-accession.  the work will now be done by reset-workspace robot (well, more specifically, by the job that robot step invokes in DSA once the corresponding DSA changes are done)